utils/exchange-bootstrap.sh

```diff
@@ -38,6 +41,23 @@ REPO_DIR="/tmp/${REPO_NAME}"
 REPO_URL="https://${USERNAME}:${PASSWORD}@github.com/${EXCHANGE_ORG}/${REPO_NAME}"
 ALIAS_URL="https://${USERNAME}:${PASSWORD}@github.com/${EXCHANGE_ORG}/${REPO_ALIAS}"
 
+# Generating GitHub PAT (personal access tokens) can no longer be done via the API
+# broken: https://developer.github.com/changes/2/#--deprecating-oauth-authorization-api
```

Comment:
We can probably reuse `tools/reset_github_user_token_and_update_circleci.sh` here once #107 is merged.
Something like this:

`tools/reset_github_user_token_and_update_circleci.sh --set-user ${PACK}`